### PR TITLE
Chore/162727542/convert database to postgres

### PR DESCRIPTION
**Pull request by habibaudu**
#### What does this PR do?
Convert mongo database to postgres and sequelize

#### Description of Task to be completed?
- create migrations folder and its migration file
- create controllers to register a user, log in using a passport, get User and update user
- change index.js to app.js
- create user model
- create config.json to configure Postgres database 
#### How should this be manually tested?
- fire up postman
   url : POST http://localhost:3000/api/v1/users
        email, username, password
   url : get http://localhost:3000/api/v1/user/:userId
        
   
#### Any background context you want to provide?
   None
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162727542
### Screenshots (if appropriate)
None
#### Questions:None

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import express from 'express';
+import bodyParser from 'body-parser';
```
Comment (oaboluwarin) [inline app.js]: Uninstall the `bodyParser` module.

Express now incorporates the `bodyParser` functionality itself. As such, the following will become:

`app.use(express.urlencoded({ extended: false }));`
`app.use(express.json());`

Comment (habibaudu) [reply app.js]: Okay. will do that now

**Comment by oreoluwade** [comment]
Create a file `server.js` in the root directory. It is in this file that you should start the server. In essence, the `app` should be imported in there and these lines should be there:


`const server = app.listen(process.env.PORT || 3000, () => {
  console.log('Listening on port ' + server.address().port);
});`

File: app.js (hunk 1 above)
Comment (habibaudu) [reply app.js]: I have uninstall the body-parser

**Review by oreoluwade — APPROVED** [review]

**Pull request merged**